db/data_migrations/20210902085638_hyphen_when_no_prison_number_letter_template.rb

```diff
@@ -0,0 +1,7 @@
+class HyphenWhenNoPrisonNumberLetterTemplate < ActiveRecord::DataMigration
```

Comment:
Not sure what this does tbh :), looks different from 20190930145607_seed_initial_letter_templates.rb. I'm assuming it's just reseeding templates but do we need that? Just a question.

Reply:
@digitalali-moj,  if just for this step here, yes, I am trying to reseeding the whole templates as I change quite a few letters.    The reason why I use this way to  update the templates is blow 
https://dsdmoj.atlassian.net/browse/CT-3777

Basically,  in the past, we often (at least me) forgot to run the reseeding command for letter when we deploy the change into different envs which is not ideal,  the main reason why we keep forgetting is that the step of updating the template is not part of auto-deployment, if we want to update, we have to login the pod and run it manually

but I have my concerns (I wrote in the above ticket) to auto it as part of deployment job, so I use data-migration to auto it,  if you think about it again,  data-migration is just right for the purpose of changing data, and also it gives your flexibility to do something more like deleting some templates not just for reseeding
let me know, if you have other thoughts, thank you!